pkg/cmd/diagnostics/diagnostics_download.go

```diff
@@ -18,7 +18,14 @@ func newDiagnosticsDownloadCommand() *cobra.Command {
 		Short: "Download diagnostics bundle",
 		Args:  cobra.MaximumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			c := pluginutil.HTTPClient("")
+			cluster, err := ctx.Cluster()
+			if err != nil {
+				return fmt.Errorf("could not get cluster: %s", err)
```

Comment:
nice. all other places went with `return nil, err`, `return 0, err` or `return "", err`. 
i like a error message to grep for! 👍

Reply:
It's good practice to add context to error and never return raw `err`